.chloggen/634.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: messaging
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add gcp_pubsub unary pull example
```

Comment:
this does not seem to be accurate, could you please update?

Reply:
I'm not sure what you mean, I do think this is accurate. I am adding a gcp pubsub unary pull example

Reply:
It introduces a bunch of new attributes and I would expect to see a description that reflects it.

Reply:
Ack, added that to the notes